src/sentry/runner/commands/devservices.py

```diff
@@ -640,12 +647,33 @@ def check_redis(containers: dict[str, Any]) -> None:
     )
 
 
+def check_zookeeper(containers: dict[str, Any]) -> None:
+    options = containers["zookeeper"]
+    port = options["environment"]["ZOOKEEPER_CLIENT_PORT"]
+    subprocess.run(
+        (
+            "docker",
+            "exec",
+            options["name"],
+            "bash",
+            "-c",
+            f"echo ruok | nc localhost {port}",
```

Comment:
this could be `'nc', 'localhost', port` and `stdin=b'ruok\n'`

Reply:
`stdin=b'ruok\n'` doesn't seem to work, I end up with the error: `'bytes' object has no attribute 'fileno'`.

I tried using os.pipe() to write the `ruok\n` value and set `stdin` to the pipe but I end up with `Command '('docker', 'exec', 'sentry_zookeeper', 'bash', '-c', 'nc', 'localhost', '2181')' returned non-zero exit status 2.`

I tried using input with no success either.

My hunch is because the pipe in this PR is happening in the docker container (i.e. bash running in the container) whereas the python pipe is going to the docker exec call (if that makes sense).

Reply:
oh it probably needs to be `input=b'ruok\n'` -- I always mix these up

Reply:
you'd remove `bash -c` as well -- no shell needed

Reply:
Ok, that worked, I'm guessing docker exec knows to pipe through to the command being run in the container?

Also I passed in the string to `input=`, otherwise I got an error.